ui/src/app/applications/components/application-urls.tsx

```diff
@@ -7,17 +7,18 @@ export const ApplicationURLs = ({urls}: {urls: string[]}) => {
         ((urls || []).length > 0 && (
             <span>
                 <a
+                    title={urls[0].split('|', 2)[0]}
                     onClick={e => {
                         e.stopPropagation();
-                        window.open(urls[0]);
+                        window.open(urls[0].split('|', 2)[1]);
```

Comment:
External URLs of Ingress/Service object won't have `|`. Please handle the case when URL does not have title.

Reply:
good point. Fixed!